brands/tourism/hotel.json

```diff
@@ -520,6 +520,21 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|TownePlace Suites": {
+    "countryCodes": ["ca", "us"],
+    "matchNames": [
+      "TownePlace Suites Marriott"
```

Comment:
There should be a comma at the end of this line. It's causing travis failures

Reply:
Thanks!  Just added it